client/src/components/friend-card.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Card } from 'antd';
+import './post-card.scss';
```

Comment:
They look weird without padding
`post-card.scss` won't work in `profile-card`

Create a new `scss` file with the following content

```css
.friend-card.ant-card {
    border-bottom: 1px solid #e8e8e8;
    cursor: pointer;

    .ant-card-body:hover {
        background: #f3f7f9;
    }
}
```